### PR TITLE
Add load test for iis container

### PR DESCRIPTION
**Pull request by YangLu1031**
1. Added a `windows-tests` folder to contain all windows yaml configs, and planning to deprecate the windows test configs in `node-throughput` folder later.
2. Reference to the pre-release fast version of wmi_exporter, so decrease the scrape interval and timeout, also change to `CL2_` variables for easier update in test definitions.
3. Bumped the pause image version

### CODE REVIEW

File: clusterloader2/testing/node-throughput/windows_override.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 WINDOWS_NODE_TEST: true
 POD_COUNT: 80
 POD_THROUGHPUT: 0.03
-CONTAINER_IMAGE: gcr.io/gke-release/pause-win:1.1.0
+CONTAINER_IMAGE: gcr.io/gke-release/pause-win:1.2.0
```
Comment (pjh) [inline clusterloader2/testing/node-throughput/windows_override.yaml]: Please change the label to 1.2.1 - we're pushing another version of this today.

File: clusterloader2/pkg/prometheus/manifests/prometheus-windows-scrape-configs.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ metadata:
 stringData:
   windows-scrape-configs.yaml: |-
     - job_name: "monitoring/windows_static"
-      scrape_interval: {{DefaultParam .CL2_WINDOWS_SCRAPE_INTERVAL "130s"}}
-      scrape_timeout: {{DefaultParam .CL2_WINDOWS_SCRAPE_TIMEOUT "120s"}}
+      scrape_interval: {{DefaultParam .CL2_WINDOWS_SCRAPE_INTERVAL "1m"}}
+      scrape_timeout: {{DefaultParam .CL2_WINDOWS_SCRAPE_TIMEOUT "10s"}}
```
Comment (pjh) [inline clusterloader2/pkg/prometheus/manifests/prometheus-windows-scrape-configs.yaml]: Consider leaving this a bit higher, just in case? Maybe 30s or 45s would still be fine.

Comment (wojtek-t) [reply clusterloader2/pkg/prometheus/manifests/prometheus-windows-scrape-configs.yaml]: 10s is still not particularly low. I think we can try 10s and fix if it will still cause issues.

File: clusterloader2/testing/node-throughput/windows_override.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
 OPERATION_TIMEOUT: 90m
 POD_STARTUP_LATENCY_THRESHOLD: 60m
-WMI_EXPORTER_URL: "https://github.com/martinlindhe/wmi_exporter/releases/download/v0.10.1/wmi_exporter-0.10.1-amd64.msi"
+WMI_EXPORTER_URL: "https://github.com/YangLu1031/wmi_exporter/releases/download/v0.10.3-alpha/wmi_exporter.exe"
```
Comment (wojtek-t) [inline clusterloader2/testing/node-throughput/windows_override.yaml]: nit: I'm assuming that at some point an official version will be relased. So how about dropping a TODO to replace with an official version once it will appear?

File: clusterloader2/pkg/prometheus/manifests/prometheus-windows-scrape-configs.yaml (hunk 2 above)
Comment (YangLu1031) [reply clusterloader2/pkg/prometheus/manifests/prometheus-windows-scrape-configs.yaml]: ok

**Comment by YangLu1031** [comment]
Thanks @wojtek-t, please take a look again?

**Comment by YangLu1031** [comment]
Add a OWNERS file under `windows-tests` folder for future management

File: clusterloader2/testing/windows-tests/config.yaml [hunk 4]
```diff
@@ -0,0 +1,91 @@
+# ASSUMPTIONS:
```
Comment (wojtek-t) [inline clusterloader2/testing/windows-tests/config.yaml]: I'm not 100% convinced that forking the config is the best option, as it increases maintainance burden, but I can probably live with it.

File: clusterloader2/testing/windows-tests/OWNERS [hunk 5]
```diff
@@ -0,0 +1,3 @@
+approvers:
+- pjh
+- YangLu1031
```
Comment (wojtek-t) [inline clusterloader2/testing/windows-tests/OWNERS]: nit: if you could add newline at the end [I'm approving to not block you - please send a separate PR for it]

Comment (wojtek-t) [reply clusterloader2/testing/windows-tests/OWNERS]: @YangLu1031 ^^
